csv-validator-core/src/main/scala/uk/gov/nationalarchives/csv/validator/schema/v1_1/SchemaParser.scala

```diff
@@ -110,12 +110,12 @@ trait SchemaParser extends SchemaParser1_0 {
   /**
     * [59] StringProvider ::= ColumnRef | StringLiteral
     */
-  override lazy val stringProvider: PackratParser[ArgProvider] = "StringProvider" ::= noext | concat | columnRef | stringLiteral ^^ {
+  override lazy val stringProvider: PackratParser[ArgProvider] = "StringProvider" ::= noExt | concat | columnRef | stringLiteral ^^ {
     s => Literal(Some(s))
   }
 
 
-  lazy val noext: PackratParser[ArgProvider] = "NoExt" ::= "noext(" ~> stringProvider <~ ")" ^^ {
+  lazy val noExt: PackratParser[ArgProvider] = "NoExt" ::= "no[eE]xt\\(".r ~> stringProvider <~ ")" ^^ {
```

Comment:
@DavidUnderdown - I'm assuming you'd rather we continue accepting 'noext' as well as 'noExt' rather than a breaking change?